Godeps/_workspace/src/github.com/GoogleCloudPlatform/kubernetes/pkg/serviceaccount/tokens_controller.go

```diff
@@ -305,6 +306,14 @@ func (e *TokensController) createSecret(serviceAccount *api.ServiceAccount) erro
 	serviceAccount.Secrets = append(serviceAccount.Secrets, api.ObjectReference{Name: secret.Name})
```

Comment:
I can pick up this bit if you want, since it's not directly related to your PR... just prevents more orphaned tokens